lang-ui/src/main/java/at/tugraz/sw20asd/lang/ui/controllers/SelectionOverview.java

```diff
@@ -88,9 +88,8 @@ public void handle(ActionEvent event) {
                 if (selectedItems.isEmpty()) {
                     updateUserInformation("nothing_selected");
                 }
-                //TODO switch to study, use selectedItems as param type ObservableList<VocabularySelectionModel>
-                //take care you get a mixed list (e.g. DE - EN and EN - DE are allowed)
-                //think about how to handle it
+                StudyVocab study = new StudyVocab(vocab, selectedItems);
```

Comment:
Should be packed in an else, because you change the page even if no entries are selected